static/app/actionCreators/group.spec.tsx

```diff
@@ -3,38 +3,39 @@ import {Client} from 'sentry/api';
 import GroupStore from 'sentry/stores/groupStore';
 
 describe('group', () => {
-  let api;
+  let api: Client;
 
   beforeEach(function () {
-    api = new Client();
+    api = new MockApiClient();
   });
 
   describe('paramsToQueryArgs()', function () {
     it('should convert itemIds properties to id array', function () {
       expect(
         paramsToQueryArgs({
-          itemIds: [1, 2, 3],
+          itemIds: ['1', '2', '3'],
           query: 'is:unresolved', // itemIds takes precedence
         })
-      ).toEqual({id: [1, 2, 3]});
+      ).toEqual({id: ['1', '2', '3']});
     });
 
     it('should extract query property if no itemIds', function () {
-      expect(
-        paramsToQueryArgs({
-          query: 'is:unresolved',
-          foo: 'bar',
-        })
-      ).toEqual({query: 'is:unresolved'});
+      const invalidArgs: any = {
+        foo: 'bar',
+      };
+
+      expect(paramsToQueryArgs({query: 'is:unresolved', ...invalidArgs})).toEqual({
+        query: 'is:unresolved',
+      });
     });
 
     it('should convert params w/o itemIds or query to empty object', function () {
-      expect(
-        paramsToQueryArgs({
-          foo: 'bar',
-          bar: 'baz', // paramsToQueryArgs ignores these
-        })
-      ).toEqual({});
+      const invalidArgs: any = {
```

Comment:
No we can't because `foo` is not an expected parameter, this test is testing something that typescript should already be validating (that you can't pass arbitrary object keys). But I don't want to remove the test __just in case__.